scripts/uploadCodeCov.js

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+const mime = require('mime-types')
+const readDir = require('recursive-readdir')
+const { S3 } = require('aws-sdk')
+
+const BUCKET_NAME = 'coverage.webdriver.io'
+const COVERAGE_DIR = path.resolve(__dirname, '..', 'coverage', 'lcov-report')
+const UPLOAD_OPTIONS = { partSize: 10 * 1024 * 1024, queueSize: 1 }
+const PR_NUMBER = process.env.TRAVIS_PULL_REQUEST // false if not PR build
```

Comment:
We want to also update the status when on master branch

Reply:
So, the [docs](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/loading-node-credentials-environment.html) say that those environment credentials should be automatically detected by the `aws-sdk` package. A [comment in the source code](https://github.com/aws/aws-sdk-js/blob/c175cb2b89576f01c08ebf39b232584e4fa2c0e0/lib/credentials/cognito_identity_credentials.js#L122) mentions that the error @mgrybyk got may mean that a region hasn't been set.

Sorry, I'm a little blind from this end because I can't see your AWS/Travis configs.

As far as updating the status, that sounds like a good idea. Could you give me some direction on that bit?

Reply:
> Could you give me some direction on that bit?

Just update the script so that it also uploads the coverage report when a push to `master` branch happened

Reply:
Ah. Okay.

Reply:
To what S3 location should master builds be pushed?

Reply:
Same but into the `master` directory. The page is accessible via http://coverage.webdriver.io/

Reply:
So we want to overwrite the contents in the master directory?